app/models/event.rb

```diff
@@ -40,6 +40,37 @@ def logo_delete
     @logo_delete || '0'
   end
 
+  def featured_projects_favorites_stats
```

Comment:
Assignment Branch Condition size for featured_projects_favorites_stats is too high. [19.13/15]